tests/kernel/mem_heap/mheap_api_concept/src/test_mheap_concept.c

```diff
@@ -133,3 +133,37 @@ void test_mheap_block_desc(void)
 		k_free(block[i]);
 	}
 }
+
+void test_mheap_block_release(void)
+{
```

Comment:
This is another situation where we seem to be duplicating test coverage that is already handled elsewhere in the file.  Doesn't test_mheap_block_desc() already do exactly this?  If it doesn't, can you maybe modify it to increase coverage instead of duplicating the whole test case?

Reply:
Thank you for your suggestion.
I think the function of test_mheap_block_release() is a little different from test_mheap_block_desc(). Because it tests the region is back to mheap pool by means of free and reallocate memory.
According to your previous suggestions, I keep allocating until it fails and use k_calloc() instead of k_malloc() to increase branch coverage from 78.6% to 85.7%.